src/sentry/static/sentry/app/views/integrationInstallation.tsx

```diff
@@ -94,9 +110,10 @@ export default class IntegrationInstallation extends AsyncView {
     const choices = this.state.organizations.map(org => [org.slug, org.slug]);
 
     const featureListHooks = HookStore.get('integrations:feature-gates');
-    featureListHooks.push(() => ({FeatureList: null}));
-
-    const {FeatureList} = featureListHooks[0]();
+    let FeatureList: FeatureListComponent | null = null;
+    if (featureListHooks.length) {
+      ({FeatureList} = featureListHooks[0]());
```

Comment:
This refactor made it a bit easier to make TS happy

Reply:
How about

```tsx
const FeatureList = featureListHooks.length
  ? featureListHooks[0]().FeatureList
  : null;
```

Reply:
In this case it should be able to infer the type and you won't need to export `FeatureListComponent`

Reply:
@EvanPurkhiser Good call on using a ternary to get TS to infer the type correctly

Reply:
Nice, did that work?

Reply:
@EvanPurkhiser Of course!